### ISSUE TITLE
Channels are labeled as Rooms

### ISSUE THREAD

**Issue by janat08** [RocketChat/Rocket.Chat#3376]
Can't find the original rooms issue that determined that a lot of code must be rewritten which must've perished once channels were labeled as rooms.

**Comment by graywolf336** [comment]
@janat08 can you provide more detail?

**Comment by janat08** [comment]
trello rooms and channels, whereas rocket appears to only have channels (rooms in rocket.chat).

**Comment by rasos** [comment]
We'd prefer labelling, as many other chat services do, even in business environments:
- private chat
- private group chat
- public group chat
Rooms / channels / conversations are confusing to our users, they do not understand the difference, until they step in and try each variant.

**Comment by janat08** [comment]
No, point in question isn't a question of labelling although youve taken the pains to adapt trello specific labelling in order to mislead the "confused users".
